src/compiler/proposals-and-stabilization/custom-test-jobs.md

```diff
@@ -0,0 +1,96 @@
+# Adding custom test jobs or adding custom test components to rust-lang/rust CI
+
+## Scope
+
+This policy is applicable to proposals for adding new test jobs or adjusting existing test jobs to build additional custom things that are *not* shipped to the end users, but may still cause [rust-lang/rust] PR CI or Full CI (the "CI") to fail, or produce a failure message, which may impact compiler/library/tools contributors who also use the [rust-lang/rust] CI.
+
+For example, this includes but is not limited to:
+
+- Ecosystem test jobs, such as Rust for Linux or Fuchsia;
+- Custom codegen backend test components, such as `codegen_gcc`, which are not part of the "main" distributed toolchain.
```

Comment:
Discussion: I don't really have a good short phrase to describe "custom test job/component", please do lmw if anyone has ideas.

Reply:
maybe "ecosystem and integration testing"?

Reply:
Changed terminology to stick with "ecosystem/integration test jobs/components" in (round 2). Filename was not renamed for review purposes.